globalmap_pcd: "$(find hdl_localization)/data/map_wuxi.pcd"
globalmap_frame_id: "map"
convert_utm_to_local: true
downsample_resolution: 0.1
use_global_localization: true
odom_child_frame_id: "ouster"
use_imu: false
invert_acc: false
invert_gyro: false
cool_time_duration: 2.0
enable_robot_odometry_prediction: false
robot_odom_frame_id: "odom"
reg_method: "NDT_OMP"
ndt_neighbor_search_method: "DIRECT7"
ndt_neighbor_search_radius: 2.0
ndt_resolution: 1.0
downsample_resolution_scan: 0.1
specify_init_pose: true
init_pos_x: 0.0
init_pos_y: 0.0
init_pos_z: 0.0
init_ori_w: 0.383
init_ori_x: 0.0
init_ori_y: 0.0
init_ori_z: -0.924
